modules/tracker/models/base/TrendModelBase.php

```diff
@@ -37,6 +37,7 @@ public function __construct()
             'config_item_id' => array('type' => MIDAS_DATA),
             'test_dataset_id' => array('type' => MIDAS_DATA),
             'truth_dataset_id' => array('type' => MIDAS_DATA),
+            'is_key_metric' => array('type' => MIDAS_DATA),
```

Comment:
Is this documented in the DAO class? I would be inclined to call this `key_metric` and add a function isKeyMetric to the DAO, as for `admin` in [UserDao](https://github.com/midasplatform/Midas/blob/master/core/models/dao/UserDao.php).